src/protocol/analysisManager.ts

```diff
@@ -166,6 +167,7 @@ class AnalysisManager {
   private readonly onAnalysisPoints = ({ analysisId, points }: analysisPoints) => {
     const handler = this.handlers.get(analysisId);
     assert(handler, "no handler for given analysisId");
+    points.map(point => ThreadFront.updateMappedLocation(point.frame));
```

Comment:
:hand: Semantically, a `.forEach()` is the right loop here, but not worth blocking the PR over :)